Ux update

OppNHeimer opened the pull request:
Added some UX and minor UI updates to allow users to browse records without interrupting audio.

san3091 commented:
Code looks good. Ran the branch locally and found a few issues.

1) The load sequence in both Chrome and Firefox is pretty chaotic and could use some smoothing out. 

2)More importantly, in Chrome specifically the player twitches back and forth when closed. It doesn't do that when it's open but it still bounces when it's closed again.

![ux-load-and-bounce](https://user-images.githubusercontent.com/14315410/97053887-c2394700-1551-11eb-8e4d-fdead2b369ec.gif)

3) I think the padding is off in the player content. Adding some space between the content and the native browser scrollbar would free up that area quite a bit.

I think we need to patch the bounce issue in this PR but I'm comfortable closing this PR after that and having separate PRs for the load sequence and the padding in the player content.

san3091 commented:
Here is the screenshot of the player as it is displayed in Chrome on a 15" mbp for reference to the padding issue.

<img width="734" alt="Screen Shot 2020-10-23 at 5 06 22 PM" src="https://user-images.githubusercontent.com/14315410/97054301-8f438300-1552-11eb-9989-efaa54e65dee.png">

OppNHeimer commented:
good catches, I fixed some funkiness with the main section width and converted the player's margin to padding which resolved the scroll bar weirdness. I didn't see the jittering when I ran in chrome, hoping it was resolved with the other changes. Let me know if it persists :)
